example/android/gradle.properties

```diff
@@ -34,5 +34,9 @@ android.enableJetifier=true
 # Enable Fabric at runtime.
 #USE_FABRIC=1
 
+# Enable new architecture, i.e. Fabric + TurboModule - implies USE_FABRIC=1.
+# Note that this is incompatible with web debugging.
+#newArchEnabled=true
```

Comment:
```suggestion
# newArchEnabled=true
```
nit - sorry it was bothering me a lot 🤣

Reply:
This line isn't a comment though. It's a setting that's commented out 😄